Storage/SQL/SQLiteBookmarksSyncing.swift

```diff
@@ -646,6 +654,22 @@ extension SQLiteBookmarks {
         return self.db.queryReturnsNoResults("SELECT 1 FROM \(TableBookmarksLocal)")
     }
 
+    public func getLocalBookmarksAdditions() -> Deferred<Maybe<[GUID: BookmarkMirrorItem]>> {
+        let sql =
+        "SELECT * FROM \(TableBookmarksLocal) AS bookmarks " +
+        "WHERE type = \(BookmarkNodeType.bookmark.rawValue) AND sync_status = \(SyncStatus.new.rawValue) AND parentID = ?"
```

Comment:
It's possible that a record exists both locally and remotely. Rather than trying to handle this case here, where you're explicitly trying to avoid merge logic, you should add:

```
AND NOT EXISTS (SELECT 1 FROM \(TableBookmarksBuffer) buf WHERE buf.guid = bookmarks.guid)
```